nominal/core/client.py

```diff
@@ -612,10 +690,12 @@ def _create_search_runs_query(
 ) -> scout_run_api.SearchQuery:
     queries = []
     if start is not None:
-        q = scout_run_api.SearchQuery(start_time_inclusive=_SecondsNanos.from_flexible(start).to_scout_run_api())
+        q = scout_run_api.SearchQuery(
```

Comment:
it looks like you're using a different set of formatting rules. you can run `just fix` to correct the formatting. Set up `ruff` as your formatter in your IDE.